admin-dev/themes/new-theme/scss/components/_grid.scss

```
@@ -93,6 +93,9 @@ table .column-filters td:last-child .grid-reset-button {
             display: inline-block;
             width: 60px;
           }
+          .d-inline-block.md-checkbox ~ .d-inline-block.form-control {
```

Can you try to select it using another class? It's dirty to select by a modifier class, and anyway these classes should'nt exist...
```suggestion
          .md-checkbox ~ .form-control {
```

Isn't that enough ?

---

They can't not exist because they are in Prestashop-Ui-Kit because it is base on Bootstrap.

---

But you can add classes to components, even if they are from the UIKit ?

---

But maybe there is a reason why it's applied only in `.d-inline-block` maybe there are several ways to display a `.md-checkbox` Sometines it's inline, sometimes not and it makes sense to have different display no?